src/utilities/EditorContext.ts

```diff
@@ -0,0 +1,13 @@
+import * as vscode from "vscode";
```

Comment:
Just a little note; importing * instead of what you are using only will make minifying in the future easy-er.

Reply:
fixed for all internally-used functions